tabs/thredds_frame_source.py

```diff
@@ -17,8 +17,8 @@
 # Data File
 TABS_DATA_URI = 'http://barataria.tamu.edu:8080/thredds/dodsC/NcML/txla_nesting6.nc'  # noqa
 CACHE_DATA_URI = os.path.join(os.path.dirname(__file__),
-                              "static/data/txla_nesting6.nc")
-DEFAULT_DATA_URI = TABS_DATA_URI
```

Comment:
I think we should default to the http connection. It would be nice to make this more flexible for testing, though.